src/components/Collapsible/style.css

```diff
@@ -10,13 +12,13 @@
   background-repeat: no-repeat;
   content: '';
   display: inline-block;
-  height: 6px;
-  margin-left: 5px;
+  height: 7*@unit;
```

Comment:
Why did you add 1 to all the values in the file?

Reply:
The `Collapsible` component is only used in 1 place, and it's in a place with `font-size: 14px` :sob:

I set new values that make the component look the same in that context